frontend/src/pages/AccountManagement/MyGroups.js

```diff
@@ -206,34 +332,111 @@ export default function MyGroups({ match }) {
+            <div className="margin-top-3 display-flex flex-align-end flex-align-center">
+              <Controller
+                control={control}
+                name={GROUP_FIELD_NAMES.IS_PRIVATE}
+                render={({ onChange: controllerOnChange, value }) => (
+                  <>
+                    <Checkbox
+                      id={GROUP_FIELD_NAMES.IS_PRIVATE}
+                      name={GROUP_FIELD_NAMES.IS_PRIVATE}
+                      className="margin-0"
+                      onChange={(e) => controllerOnChange(e.target.checked)}
+                      label="Keep this group private."
+                      checked={value}
+                      disabled={isAppLoading}
+                    />
+                    <QuestionTooltip text="Keep this group private or uncheck to make public for your region" />
+                  </>
+                )}
+              />
+            </div>
+            {!watchIsPrivate && (
+              <div className="margin-top-4">
+                <label className="display-block margin-bottom-1">
+                  Add co-owner
+                  {' '}
+                  <Req />
 
-          <div className="margin-top-4 display-flex flex-align-end">
-            <Controller
-              control={control}
-              name={GROUP_FIELD_NAMES.IS_PRIVATE}
-              render={({ onChange: controllerOnChange, value }) => (
-                <>
-                  <Checkbox
-                    id={GROUP_FIELD_NAMES.IS_PRIVATE}
-                    name={GROUP_FIELD_NAMES.IS_PRIVATE}
-                    className="margin-0"
-                    onChange={(e) => controllerOnChange(e.target.checked)}
-                    label="Keep this group private."
-                    checked={value}
+                  {coOwnerError && <span className="usa-error-message">{coOwnerError.message}</span>}
+                  <div>
+                    <span className="usa-hint">
+                      Choose up to 3 co-owners who can change permissions and edit the group.
+                    </span>
+                  </div>
+                  <MultiSelect
+                    name={GROUP_FIELD_NAMES.CO_OWNERS}
+                    options={filterCoOwnerOptions()}
+                    control={control}
+                    simple={false}
+                    required="Select at least one"
                     disabled={isAppLoading}
+                    rules={maxCoOwnerRule}
+                  />
+                </label>
+                <div className="margin-top-4">
+                  <label htmlFor={GROUP_FIELD_NAMES.NAME} className="display-block margin-bottom-1">
+                    Who do you want to share this group with?
+                    <QuestionTooltip text="Shared groups can be seen and used by others but only you and co-owners can edit the group." />
+                  </label>
+                  <Controller
+                    control={control}
```

Comment:
I am able to submit the form without selecting anything for this field

Reply:
If you are talking about the radio buttons you shouldn't have to select anything but it will default to 'everyone' the next time you edit the group.

I also made a recent change that no longer requires co-owners.

Reply:
If there's a default - 'everyone' - the radio buttons should have that selected as a default value.

Reply:
Kelly was very adamant about having neither option selected by default.

Reply:
But I agree with you.